src/lib.rs

```diff
@@ -568,24 +582,37 @@ impl VectorSelector {
     }
 }
 
-extension_sql!(
-    r#"
-CREATE AGGREGATE vector_selector(
-    start_time timestamptz,
-    end_time timestamptz,
-    bucket_width bigint,
-    lookback bigint,
-    sample_time timestamptz,
-    sample_value DOUBLE PRECISION)
-(
-    sfunc = vector_selector_transition,
-    stype = internal,
-    finalfunc = vector_selector_final,
-    combinefunc = vector_selector_combine,
-    serialfunc = vector_selector_serialize,
-    deserialfunc = vector_selector_deserialize,
-    parallel = safe
-);
-"#,
-    name = "create_aggregate_vector_selector"
-);
+// extension_sql!(
```

Comment:
The answer to this question (and a few others that you've asked) is that this PR is not a complete picture of all of the changes which need to be merged to get schema generation working. I wanted to have some eyes on the work that I'm doing and get early feedback without overwhelming reviewers with a code dump.

I commented out this code in this change to allow the extension to be built and installed. In a follow-on change it will be un-commented and tested.